examples/gui/samplegraph/mainwindow.cpp

```diff
@@ -169,7 +169,7 @@ void MainWindow::generateSampleData()
 		}
 		else if(ch->shvPath() == "Discrete") {
 			style.setInterpolation(tl::GraphChannel::Style::Interpolation::None);
-			style.setDrawDiscreteValuesInfo(true);
+			// style.setHideDiscreteValuesInfo(false);
```

Comment:
whyyyyyyyyyyyyyyyyyyyyyyyy

Reply:
It is example code, I like to show, that there is also other option.